### PR TITLE
[10.0] mig hr employee current contract mgs

### PR DESCRIPTION
**Pull request by GillesTephaneMeyomesse**
migration v8 -> v10

### CODE REVIEW

File: hr_employee_current_contract/models/hr_contract.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, exceptions, _
```
Comment (adrienpeiffer) [inline hr_employee_current_contract/models/hr_contract.py]: Replace openerp by odoo

File: hr_employee_current_contract/models/hr_employee.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
```
Comment (adrienpeiffer) [inline hr_employee_current_contract/models/hr_employee.py]: same comment

File: hr_employee_current_contract/tests/test_hr_employee_current_contract.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests import common
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+from datetime import datetime, timedelta
+from openerp import exceptions
```
Comment (adrienpeiffer) [inline hr_employee_current_contract/tests/test_hr_employee_current_contract.py]: same comment

File: hr_employee_current_contract/views/hr_contract_view.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
```
Comment (adrienpeiffer) [inline hr_employee_current_contract/views/hr_contract_view.xml]: Same comment

File: setup/hr_employee_current_contract/.eggs/README.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```
Comment (adrienpeiffer) [inline setup/hr_employee_current_contract/.eggs/README.txt]: I think that we should not commit .eggs ?
So, you can add a line in .gitignore for that

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Comment by adrienpeiffer** [comment]
@GillesTephaneMeyomesse Can you try to rebase your branch on the 10.0 branch ? for the travis build.

File: setup/hr_employee_current_contract/.eggs/README.txt (hunk 5 above)
Comment (GillesTephaneMeyomesse) [reply setup/hr_employee_current_contract/.eggs/README.txt]: I think I have corrected your remarks =) 
I added .eggs* in .gitignore file

File: .gitignore [hunk 6]
```diff
@@ -54,3 +54,6 @@ docs/_build/
 # Backup files
 *~
 *.swp
+
+# eggs
+.eggs*
```
Comment (adrienpeiffer) [inline .gitignore]: I think that the star isn't necessary ?

**Pull request closed without merging**